### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Success Criterion 2.4.13: Page Break Navigation (A)

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1893]
“Success Criterion 2.4.13 Fixed Reference Points (Level A): For web content with page break locators, a mechanism is available to navigate to each locator.”

We kindly request significant changes to this SC and its name, which we believe to be more vague than the previous title. The updated SC title, normative part of the SC, Intent, and Benefits don't explicitly reference ePub. The first reference isn't made until the Example section. We agree with the benefit but think this would be much more powerful combined with an existing criteria, to encourage people to add reference points to content that allows navigation on all content beyond ePubs. 

In the Benefits section we also wonder if it would be helpful to specify that the two "types" of page numbers (electronic version & default view/printed version) should be clearly distinguished, particularly to help SR users?

Other than that, our UX writer also identified a few minor typos and ungrammatical text in the new text. Would it be helpful to send those over or is that not needed at this stage?

**Comment by mraccess77** [comment]
I'd also add that it's my understanding that the page break locator's don't need to be programmatically communicated - just that they can be reached with navigation.  However, there could be an intersection here with other SC such as SC 1.3.1 Info and Relationships that may come into play when they exist visually.   I personally agree that knowing the corresponding page numbers in print to the digital version is absolutely needed for people.  In particular students who use accessible digital content need more than navigation to pages but also need to know the corresponding page numbers in order to complete assignments or research.  Frankly this issue extends into audio content as well as many of our audio book readers have no good way to correlate their location back to the print version which is required in an education setting.

**Comment by alastc** [comment]
> The updated SC title, normative part of the SC, Intent, and Benefits don't explicitly reference ePub.

Writing in technology agnostic terms is part of the [requirements for WCAG 2.x](https://www.w3.org/TR/wcag2-req/#cross-tech). Any technology scoping has to be done by terms such as "in markup languages" or similar ways.

> We agree with the benefit but think this would be much more powerful combined with an existing criteria, to encourage people to add reference points to content that allows navigation on all content beyond ePubs

It would be more powerful, but there is a significant concern that it could be over-reaching. For example, should every document with headings have a table of contents? How would you define when it should or should not be included?

Editorial updates are welcome.

**Comment by alastc** [comment]
@aubbink1 - is there anything we can add in the 1st paragraph to prevent this type of misunderstanding?

**Comment by alastc** [comment]
The response above was agreed: https://www.w3.org/2021/08/17-ag-minutes.html#t13

If there are any further editorial updates, please do let me know, thank you.
